### PR TITLE
Add change views

### PR DESCRIPTION
**Pull request by imsickofmaps**

### CODE REVIEW

**Comment by imsickofmaps** [comment]
@gsvr review please

File: ci/views.py [hunk 1]
```diff
@@ -325,10 +326,11 @@ def identity(request, identity):
             pass
         else:
             results = idApi.get_identity(identity)
-            reg_filter = {
+            hub_filter = {
                 "mother_id": identity
             }
-            registrations = hubApi.get_registrations(params=reg_filter)
+            registrations = hubApi.get_registrations(params=hub_filter)
+            changes = hubApi.get_changess(params=hub_filter)
```
Comment (gsvr) [inline ci/views.py]: Shouldn't this be get_changesss? :stuck_out_tongue_winking_eye:

**Comment by gsvr** [comment]
Apart from that one comment it looks okay, but travis is failing.

**Comment by gsvr** [comment]
:+1:

**Pull request merged**